src/components/pages/home/elements/WeatherBlock.js

```diff
@@ -20,10 +24,18 @@ export default class WeatherBlock extends Component {
     getMarsWeatherData()
       .then(weather => (this.setState({
         weather: weather.slice(weather.length - 1)[0],
-      })));
+      }))).catch(() => (this.setState({ error: true })));
   }
 
   render() {
+    if (this.state.error === true) {
```

Comment:
We don't need to compare boolean values to `true`, `if (this.state.error)` is logically equivalent.